### PR TITLE
adding libdbus-1-dev to dockerfile

### PR DESCRIPTION
**Pull request by briancorbin**
Dupe of #33

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,26 +16,27 @@ RUN  ln -fs /usr/share/zoneinfo/Etc/UTC /etc/localtime \
   && apt-get update \
   && apt-get upgrade -y \
   && apt-get install -y \
-     build-essential \
```
Comment (jgreat) [inline Dockerfile:19]: Can we reset the white space in this PR please.

**Review by jgreat — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (briancorbin) [reply Dockerfile]: should be fixed!

File: Dockerfile [hunk 2]
```diff
@@ -23,6 +23,7 @@ RUN  ln -fs /usr/share/zoneinfo/Etc/UTC /etc/localtime \
      git \
      jq \
      libclang-dev \
+     libdbus-1-dev \
```
Comment (jgreat) [inline Dockerfile:26]: Can you add a note in the comments above this section on what we need this new requirement for.  The idea is to minimize our dependencies, so tracking their purpose is really helpful.

**Review by jgreat — DISMISSED** [review]
Approved, but would like it if we can update the comment above.

File: Dockerfile (hunk 2 above)
Comment (briancorbin) [reply Dockerfile:30]: Does this change seem sufficient?

**Review by jgreat — APPROVED** [review]

**Pull request merged**